apps/challenges/urls.py

```diff
@@ -31,4 +31,14 @@
     url(r'(?P<challenge_pk>[0-9]+)/challenge_phase/(?P<challenge_phase_pk>[0-9]+)/submissions$',
         views.get_all_submissions_of_challenge,
         name='get_all_submissions_of_challenge'),
+    url(r'challenge/challenge_host_team/(?P<challenge_host_team_pk>[0-9]+)/step_1/$',
```

Comment:
Lets prepend `challenge/create` instead of `challenge/` to all the challenge creation URLs.